### DISCUSSION TITLE
Welcome to lxqt Discussions!

### DISCUSSION THREAD

**Discussion by tsujan** [lxqt/lxqt#1872] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tsujan** [comment]
OK, let's discuss about GitHub's "Discussions" here!

As an LXQt member, I was able to open this page. For starters, I have 3 questions:

 1. Do other members get notified about it?
 2. Can users see and participate in it?
 3. Can users open a discussion?

**Reply by stefonarch** [reply]
1) I got an email
```
 [lxqt/lxqt] Welcome to lxqt Discussions! (#1872)
You are receiving this because you are subscribed to this thread.
```
2-3) to test, there should be some settings for levels of permissions.

**Reply by ito32bit** [reply]
>     3\. Can users open a discussion?

My browser:
![LXQt_Discussions_screen-2020-12-09_19-36-57](https://user-images.githubusercontent.com/52656419/101619353-7f2a2900-3a56-11eb-80e7-c5e44bac6f4d.png)

**Reply by stefonarch** [reply]
* I see here on the right:

```
Lock conversation (not tested)
Transfer discussion (no repo found)
Pin discussion (works, also unpin)
```
* answering own questions is not possible ;)
* no settings accessible for add categories and so on.
* labels are different somehow (maintainer, collaborator) as in the repos, those are probably in the settings.
* edit titles is allowed for me

**Reply by tsujan** [reply]
I think it can be used like the forum we had.

**Reply by tsujan** [reply]
> * no settings accessible for add categories and so on.

There is :) Categories can be added and removed.
